platform/linux-generic/odp_schedule_basic.c

```diff
@@ -205,9 +205,9 @@ typedef struct {
 	uint8_t          load_balance;
 	uint16_t         max_spread;
 	uint32_t         ring_mask;
-	odp_spinlock_t   mask_lock;
 	odp_atomic_u32_t grp_epoch;
 	odp_shm_t        shm;
+	odp_spinlock_t   mask_lock[NUM_SCHED_GRPS];
```

Comment:
Typo in the commit message: `... which is called from in the fast path ...`

Reply:
fixed in v6